Implement Discord support (#5)
smogon/GitHub-Bot: README.md

GitHub Bot
==========

> This bot receives commit data from GitHub, and posts in the Showdown chat, with the option of a Discord server.

Reviewer: ```suggestion
This bot receives commit data from GitHub and can be configured to post in Showdown chat and/or a Discord server.
```
I think in theory the bot should also support Discord only, but thats perhaps out of scope for the time being.

Author: 👍

-   `server` - server address (default sim.smogon.com)
-   `serverport` - server port (default 8000)
-   `serverid` - server identifier used for logins (default showdown)

> -   `token` - token used to login to discord

Reviewer: ```suggestion
-   `token` - token used to login to Discord
```
